geode-core/src/integrationTest/java/org/apache/geode/distributed/LauncherMemberMXBeanIntegrationTest.java

```diff
@@ -227,6 +227,8 @@ public void showJVMMetrics_returnsOJVMMetricsType()
         if (cdsValue.equals(osMetricValue)) {
           break;
         }
+        System.gc();
+        System.runFinalization();
```

Comment:
I'm really reluctant to add `gc()` or `runFinalization()`. Why do you think these will fix flakiness in this test?

Reply:
Yes, I'm probably on the wrong track here. The test flakiness is around reporting memory stats. Since the underlying (volatile) values are retrieved from `VMStatsMonitor` would it then be better to reduce the stat sampling period so that values are updated more frequently?

Reply:
or can you change the test to be more patient to see an expected value? Do you know how often these values are currently being updated?

Reply:
If I understand the code correctly, I believe they are being updated via a StatsMonitor, so I presume no more frequently than `statistic-sample-rate`. The test could retrieve that setting and try for at least that amount of time.